tests/gate_fixture.py

```
@@ -229,12 +247,85 @@ def _lane_manifest_entry(
+        },
+    ],
+}
+
+
 class GateFixture:
     """A six-lane curation scenario laid out under one temporary root."""
 
-    def __init__(
-        self, root, bridge_records=None, thalamic_records=None, *, canonical_ids=True
-    ):
+    def __init__(self, root, bridge_records=None, thalamic_records=None, *, canonical_ids=True):
```

Ruff (0.16.2)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)